src/libraries/System.Configuration.ConfigurationManager/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -0,0 +1,161 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="System.Configuration.ConfigurationManager, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51">
```

Comment:
ConfigurationManager is not trimmable. See [https://github.com/dotnet/runtime/pull/49085](https://github.com/dotnet/runtime/pull/49085). We probably don't need to waste time analyzing this assembly until we try to make it trimmable (which isn't in the near future).